components/tensorflow-notebook-image/install.sh

```diff
@@ -46,7 +46,7 @@ jupyter labextension install @jupyter-widgets/jupyterlab-manager
 
 # Install common packages from requirements.txt for both python2 and python
 pip --no-cache-dir install -r /tmp/requirements.txt
-source activate py2
+source activate base
```

Comment:
So is the problem that source activate py2 has no effect because that's already the default?

What is the base environment?

Reply:
```source activate py2``` is done [here](https://github.com/kubeflow/kubeflow/blob/master/components/tensorflow-notebook-image/install.sh#L7) already. ```base``` is python3 environment and the latter activate should be ```base``` to install the packages from requirements.txt